src/main/java/org/xbill/DNS/ResolverConfig.java

```diff
@@ -30,9 +31,11 @@
  * @author <a href="mailto:yannick@meudal.net">Yannick Meudal</a>
  * @author <a href="mailto:arnt@gulbrandsen.priv.no">Arnt Gulbrandsen</a>
  */
-
 public class ResolverConfig {
 
+	static final String DNS_SERVER_PROP = "dns.server";
```

Comment:
Made public, dropped indent (when is the mass test formatting happening?)

Reply:
Not sure yet, if no pull requests are open. And only on test sources anyway. I haven't decided to apply it on the main sources yet since they're not so messy.